books/free-programming-books-subjects.md

```diff
@@ -245,6 +246,11 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Uncomplicating Kubernetes](https://livro.descomplicandokubernetes.com.br/en/) - Jeferson Fernando
 
 
+### Data Visualization
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.
I suggest changing the 'datamining' category to 'data science' and add this there.

Reply:
good resource

Reply:
Thank you for the suggestion. By the list of the books available in the file there are no other data visualization related books found. So I will change the 'datamining' category to 'data science'.